chainer/links/caffe/caffe_function.py

```diff
@@ -318,6 +319,54 @@ def _setup_relu(self, layer):
         self.forwards[layer.name] = fw
         self._add_layer(layer)
 
+    @_layer('Eltwise', 'ELTWISE')
+    def _setup_eltwise(self, layer):
+        operation = layer.eltwise_param.operation
+        coeffs = layer.eltwise_param.coeff
+        if coeffs == []:
+            coeffs = None
+
+        self.forwards[layer.name] = _EltwiseFunction(operation, coeffs)
+        self._add_layer(layer)
+
+    @_layer('Scale', 'NONE')    # TODO: old name
```

Comment:
I think it is better to use `None` for indicating that this layer does not have old name, rather than string value. Then we should treat `None` as a special case in `_layer` method.